README.md

```diff
@@ -65,6 +66,9 @@ Quick Start
 
 ```
 
+Example is taken from [test](/src/test/java/com/scaleset/cfbuilder/TemplateTest.java).
+See `/src/test/java/com/scaleset/cfbuilder/` for more tests that you can take as examples.
```

Comment:
I'd rephrase this:
`... for more tests that you can take as examples.`
to
`... for more tests containing examples that you can use.`